handlers/prometheus_handlers.go

```diff
@@ -33,6 +34,11 @@ func (h *Handler) PrometheusConfigHandler(w http.ResponseWriter, req *http.Reque
 			http.Error(w, "unable to connect to prometheus", http.StatusInternalServerError)
 			return
 		}
+
+		if strings.Contains(promURL, "/graph") {
+			promURL = strings.TrimSuffix(promURL, "/graph")
```

Comment:
@subhamkrai you've got the right idea. 👍 Ultimately, what we'll want here is to strip off any text after the ip address and port number. So, the truncator should leave the first portion inside of the square brackets (`[]`) below alone, but remove everything after that - `[http://x.x.x.x:xxxx]/foo/bar/anything`

You'll need to avoid matching a specific string like "/graph".